src/main/java/com/conveyal/gtfs/validator/MTCValidator.java

```diff
@@ -0,0 +1,56 @@
+package com.conveyal.gtfs.validator;
+
+import com.conveyal.gtfs.error.SQLErrorStorage;
+import com.conveyal.gtfs.loader.Feed;
+import com.conveyal.gtfs.model.*;
+
+import static com.conveyal.gtfs.error.NewGTFSErrorType.FIELD_VALUE_TOO_LONG;
+
+/**
+ * MTCValidator checks in a GTFS feed that the length of certain field values
+ * do not exceed the 511 MTC guidelines. (TODO: add guidelines URL.)
+ * To refer to specific limits, search the guidelines for the word 'character'.
```

Comment:
Can you adjust this to say something like "MTCValidator runs a set of custom validation checks for GTFS feeds managed by MTC in Data Tools. " and then list the field length check as the primary check for now? Also, might be worth referring to the issue: https://github.com/ibi-group/datatools-ui/issues/544